Readme.md

```diff
@@ -61,22 +61,20 @@ play along! :tada:
 1. Clone this repo.
 2. Get [Node/NPM](https://nodejs.org) and [yarn](https://yarnpkg.com)
    (`npm install -g yarn`).
-3. Run `make` to install dependencies and compile the source files.
+3. Run `yarn` to install dependencies and compile the source files.
```

Comment:
Should remove `and compile the source files` as `yarn` only installs deps. Should then have an extra step for compiling (using either of the scripts `yarn watch`, for incremental dev builds, or `yarn build`, for standard dev builds).